### ISSUE TITLE
[BUG] segmentation fault deleting a secondary ip that does not exist of a port of a router

### ISSUE THREAD

**Issue by pinoOgni** [polycube-network/polycube#350]
## Describe the bug

Segmentation fault when trying to delete a secondaryip address in the format "ip-address/N" that does not exist.

## To Reproduce

In one terminal:
```
~  polycubectl router r1 
~  polycubectl r1 ports p1
~  polycubectl r1 ports p1
~  polycubectl r1 ports p1 set ip=10.0.0.1/24            
~  polycubectl r1 ports p1 secondaryip del 10.0.0.254/24 
Delete http://localhost:9000/polycube/v1/router/r1/ports/p1/secondaryip/10.0.0.254%2F24/: EOF
```
In another terminal:

```
~ sudo polycubed   
...
[2020-10-26 14:37:22.106] [Router] [r1] [info] Creating Router instance
[2020-10-26 14:37:33.637] [Router] [r1] [info] Added new port: p1 (index: 0) [mac: 56:21:c9:fb:d4:81 - ip: ]
[2020-10-26 14:38:33.520] [Router] [r1] [info] Added route [network: 10.0.0.1/32 - nexthop: 0.0.0.0 - interface: p1]
[2020-10-26 14:38:33.520] [Router] [r1] [info] Added route [network: 10.0.0.0/24 - nexthop: 0.0.0.0 - interface: p1]
[1]    15660 segmentation fault  sudo polycubed
```

## Expected behavior
Maybe a "warning" to the user instead of "segmentation fault". 

## Please tell us about your environment: 
1. OS details: cat /etc/-release 
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.1 LTS"
NAME="Ubuntu"
VERSION="20.04.1 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.1 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

```

2. Kernel details: uname -a
```
5.4.0-52-generic #57-Ubuntu SMP Thu Oct 15 10:57:00 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

```
3. Polycube Version: polycubectl version
```
polycubectl:
 version: v0.9.0+ [git: (branch/commit): master/79889309]
polycubed:
 version: v0.9.0+ [git: (branch/commit): master/79889309]
```

**Comment by FedeParola** [comment]
All the concept of secondary ip on the router is a bit unclear to me, for example, what is practically the difference between a primary and a secondary ip address? What happens if a port has secondary addresses and the primary address is deleted? I wonder if it would be better to remove the concept of secondary address and just have a list of ip addresses. This would also be closer to Linux behaviour, where it is possible to assign multiple addresses to an interface but there is no distinction between primary and secondary (at least as far as I know).

**Comment by frisso** [comment]
Federico, this looks a very nice proposal to me: avoid 'secondary' address, and just support a list of addresses, such as in Linux.
The concept of 'secondary' address comes from the Cisco world, as they define a single primary address, and a list of secondary addresses.

**Comment by frisso** [comment]
Closed with #385
